Custom-Range-Slider/README.md

```diff
@@ -0,0 +1,26 @@
+<h1>Custom Range Slider</h1>
+
+<p>Simple Custom Range Slider written in HTML, CSS, and JavaScript .</p>
+
+<h3>Used Technologies</h3>
+<ul>
+  <li>HTML5</li>
+  <li>CSS3</li>
+  <li>JavaScript</li>
+</ul>
+
+#### Steps to Use:
+---
+
+- Download or clone the repository
+```
+git clone https://github.com/Ayushparikh-code/Web-dev-mini-projects.git
+```
+- Go to the directory
+- Run the index.html file
+- Set the Range According to your wish
+
+
+<h3> ScreenShots </h3>  
+<img src = "Custom-Range-Slider.png" alt="Sample Image">
+<br>
```

Comment:
Apply markdown beautifier. Use link https://codebeautify.org/markdown-formatter

Also can you please put the image, the link isn't working, 
Use link
https://stackoverflow.com/questions/14494747/how-to-add-images-to-readme-md-on-github

Reply:
Done @chicken-biryani

Reply:
@Atif0604 The image is still not working. Can u please have a look into that. The rest looks good to go. Just make the image work.

Reply:
I made the some changes @chicken-biryani 
this is the bulid what i see now in my repo 
![image](https://user-images.githubusercontent.com/64218887/124365566-8caf3680-dc66-11eb-8bad-3ab95522dd93.png)